src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProjectWithP2PReference.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.Xml.Linq;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+using System.Runtime.CompilerServices;
+using NuGet.Packaging;
+using System.IO;
+using System.Collections.Generic;
+
+namespace Microsoft.NET.ToolPack.Tests
+{
+    public class GivenThatWeWantToPackAToolProjectWithP2PReference : SdkTest
+    {
+
+        public GivenThatWeWantToPackAToolProjectWithP2PReference(ITestOutputHelper log) : base(log)
+        {
+
+        }
+
+        private string SetupNuGetPackage([CallerMemberName] string callingMethod = "")
+        {
+            TestAsset testAsset = _testAssetsManager
+                .CopyTestAsset("PortableToolWithP2P", callingMethod)
+                .WithSource()
+                .WithProjectChanges(project =>
+                {
+                    XNamespace ns = project.Root.Name.Namespace;
+                    XElement propertyGroup = project.Root.Elements(ns + "PropertyGroup").First();
```

Comment:
This doesn't seem to make any changes. Is WithProjectChanges needed?

Reply:
you are right, it should be moved. Several back and forth cause it left there

Reply:
fixed